src/main/steps/c100-rebuild/other-person-details/address/lookup/AddressLookupPostController.test.ts

```diff
@@ -70,4 +70,36 @@ describe('applicant1 > address > lookup > AddressLookupPostController', () => {
 
     expect(res.redirect).toHaveBeenCalled();
   });
+
```

Comment:
Please remove this test case from here and append this test case in https://github.com/hmcts/prl-citizen-frontend/pull/585/ as the controller has been moved to common place - src/main/steps/c100-rebuild/people/LookupAndManualAddressPostController.ts